src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/UbiquitousLanguageAsFile.java

```diff
@@ -19,6 +21,7 @@
 @Accessors(fluent = true)
 public abstract class UbiquitousLanguageAsFile implements UbiquitousLanguageRepository {
 
+  public static final Path MANIFEST_DESTINATION_FOLDER = Path.of("target/candy-doc");
```

Comment:
I did that because we said that the manifest file will always be at the same place, but if the manifest file is with the other file, this variable is no longer relevant.

Reply:
You can take a look at the `forFormat` method. The destination folder is computed from the build directory and "candy-doc/ubiquitous_language". IMO, it make sense that the manifest can be in the ubiquitous_language folder OR in the candy-doc folder. We don't know the complete scope of this manifest yet. What's your opinion ?

Reply:
At the beginning, we want to put the manifest in the candy-doc folder. But if it makes sense in the ubiquitous language folder, I think we can put it here, precisely because we don't know the scope yet.